src/main/java/cash/z/ecc/android/sdk/block/CompactBlockProcessor.kt

```diff
@@ -428,18 +428,20 @@ class CompactBlockProcessor(
     internal suspend fun refreshUtxos(tAddress: String, startHeight: Int): Int? = withContext(IO) {
         var count: Int? = null
         // todo: cleanup the way that we prevent this from running excessively
-        //       For now, try for about 5 minutes, per app launch. If the service fails it is
+        //       For now, try for about 3 blocks per app launch. If the service fails it is
         //       probably disabled on ligthtwalletd, so then stop trying until the next app launch.
-        if (failedUtxoFetches < 15) { // there are approx 3 attempts per minute so 15 ~= 5min
+        if (failedUtxoFetches < 9) { // there are 3 attempts per block
```

Comment:
If this happens every 20 seconds, can't it be done once and retried again on the next poll?